ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsWriter.scala

```diff
@@ -41,20 +41,11 @@ object TransactionsWriter {
         Timed.value(deleteContractsBatch, deleteContracts.execute())
       }
 
-      for (insertContracts <- contractsTableExecutables.insertContracts) {
-        Timed.value(insertContractsBatch, insertContracts.execute())
-      }
-
-      // Insert the witnesses last to respect the foreign key constraint of the underlying storage.
-      // Compute and insert new witnesses regardless of whether the current transaction adds new
-      // contracts because it may be the case that we are only adding new witnesses to existing
-      // contracts (e.g. via divulging a contract with fetch).
-      for (insertWitnesses <- contractWitnessesTableExecutables.insertWitnesses) {
-        Timed.value(insertContractWitnessesBatch, insertWitnesses.execute())
```

Comment:
Can you add back the metrics timers for these inserts?

Reply:
And the comment, as it is useful

Reply:
fixed